pkg/cluster/config.go

```diff
@@ -296,3 +297,18 @@ func ValidateCIDR(cidr string) error {
 	}
 	return nil
 }
+
+// GetClusterDNS returns the IP address of the DNS server, given the service CIDR of the cluster
+func GetClusterDNS(iprange string) (string, error) {
```

Comment:
Please don't introduce a function in one commit and then immediately move it around in the next commit. Please handle it in the commit where you are introducing the new function.

Reply:
Sure, I plan to squash the commits. Just wanted to highlight the refactor in a separate commit for reviewing purposes.

Reply:
OK, it was not clear to me that a squash was planned given we have not been using commits to address changes requested by the reviewers.

Reply:
Fixed